src/message.js

```diff
@@ -178,6 +178,60 @@ Message.prototype.encrypt = function(keys) {
 };
 
 /**
+ * Encrypt the message symmetrically using a passphrase.
+ *   https://tools.ietf.org/html/rfc4880#section-3.7.2.2
+ * @param {String} passphrase
+ * @return {Array<module:message~Message>} new message with encrypted content
+ */
+Message.prototype.symEncrypt = function(passphrase) {
+  if (!passphrase) {
+    throw new Error('The passphrase cannot be empty!');
+  }
+
+  var algo = 'aes256';
+  var packetlist = new packet.List();
+
+  // create a Symmetric-key Encrypted Session Key (ESK)
+  var symESKPacket = new packet.SymEncryptedSessionKey();
+  symESKPacket.sessionKeyAlgorithm = algo;
+  symESKPacket.decrypt(passphrase);
```

Comment:
The `decrypt` invocation is a little confusing here, but correct using of the current `SymEncryptedSessionKey` api. See https://github.com/openpgpjs/openpgpjs/blob/master/src/packet/sym_encrypted_session_key.js#L108-L117